### PR TITLE
sBonfire 0.80

### PR DESCRIPTION
**Pull request by SD8Z**
Made the changes Timer suggested.

### CODE REVIEW

File: src/sd8z/scripts/sbonfire/SBonfire.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 import java.awt.*;
 
-@Script.Manifest(name = "sBonfire", description = "Makes bonfires for firemaking xp.", properties = "version=0.71;topic=1136177")
+@Script.Manifest(name = "sBonfire", description = "Makes bonfires for firemaking xp.", properties = "version=0.80;topic=1136177")
```
Comment (Timer) [inline src/sd8z/scripts/sbonfire/SBonfire.java]: All you're missing now is your author tag and topic id!

Comment (SD8Z) [inline src/sd8z/scripts/sbonfire/SBonfire.java]: Topic there is.
Wasn't the author tag removed?

Comment (Timer) [inline src/sd8z/scripts/sbonfire/SBonfire.java]: Nope, in properties put author=SD8Z;

Comment (Timer) [inline src/sd8z/scripts/sbonfire/SBonfire.java]: and the trailing `;` on topic, its bugging me.

Comment (SD8Z) [inline src/sd8z/scripts/sbonfire/SBonfire.java]: Oh.
Added.

**Comment by Strikegently** [comment]
Read the instructions and try again.

**Scripts must have a `topic` tag with a thread ID corresponding to your script thread in the Projects section**

**Comment by Timer** [comment]
@SD8Z what he means is please create a new topic and update your topic id accordingly

**Comment by SD8Z** [comment]
Can't the thread be moved though? Or is creating a new one mandatory?

**Comment by Timer** [comment]
@SD8Z that thread is permanently stuck in place, we can't move it -- sorry!

**Comment by SD8Z** [comment]
Alright.

**Comment by Timer** [comment]
@SD8Z great!

**Comment by Timer** [comment]
revert wat @SD8Z you just removed someone else's script

**Comment by SD8Z** [comment]
Woops

**Comment by SD8Z** [comment]
Woops

**Pull request closed without merging**